/*
        *** JavaScript try and catch ***
    - The try statement allows you to define a block of code to be tested for errors while
    it is being executed.
    - The catch statement allows you to define a block of code to be executed, if an error
    occurs in the try block.
    - The JavaScript statements try and catch come in pairs: */
    
try {
    //Block of code to try
}
catch(err) {
    //Block of code to handle errors
}
    
/*  When an error occurs, JavaScript will normally stop and generate an error message.
    - The technical term for this is: JavaScript will throw an exception (create an Error
    object with two properties: name and message) */
